lib/public/Http/Client/IClientService.php

```diff
@@ -35,8 +37,14 @@
  */
 interface IClientService {
 	/**
+	 * @param CookieJarInterface $cookieJar
 	 * @return IClient
 	 * @since 8.1.0
 	 */
-	public function newClient(): IClient;
+	public function newClient(?CookieJarInterface $cookieJar): IClient;
```

Comment:
```suggestion
	public function newClient(?CookieJarInterface $cookieJar = null): IClient;
```

This is needed so we don't break old calls without any arguments